app/main.py

```diff
@@ -0,0 +1,187 @@
+import json
+import os
+import sys
+
+from pathlib import Path
+
+sys.path.append(os.getcwd())
```

Comment:
Avoid using this line, it's not good to modify sys variables on-the-fly.
Please, describe your problem, I think there is a better solution exist

Reply:
Main problem is then i start my streamlit app using ```streanlit run app/main.py``` , it has no project directory in paths, so i cant import ```src``` (from project repository). Here is ```sys.path``` value:
with error ```ModuleNotFoundError: No module named 'src'```.
```
[
0:"/home/jennsy/PycharmProjects/configNLP/app/main.py"
1:"app"
2:"/home/jennsy/PycharmProjects/configNLP/venv/lib/python3.10/site-packages/git/ext/gitdb"
3:"/home/jennsy/PycharmProjects/configNLP/venv/bin"
4:"/usr/lib/python310.zip"
5:"/usr/lib/python3.10"
6:"/usr/lib/python3.10/lib-dynload"
7:"/app/lib/python3.10/site-packages"
8:"/home/jennsy/PycharmProjects/configNLP/venv/lib/python3.10/site-packages"
]
```
And with ```append``` it appears:
```
...
7:"/app/lib/python3.10/site-packages"
8:"/home/jennsy/PycharmProjects/configNLP/venv/lib/python3.10/site-packages"
9:"/home/jennsy/PycharmProjects/configNLP"
]
```

Sorry, it was first working solution googled, that at least work. But now i can see it appends every time script runned. So i added check. I hope for your alternatives! =)  Or mb check will be enough? Things getting complicated then i deploy.